lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
just trying to reduce the number of changes in UI between minor versions, specially in a timeslot between RC releases. However I agree that most users do not call this.